### ISSUE TITLE
[Bug]: Fail to launch any exe by shortcut (context menu in file explorer)

### ISSUE THREAD

**Issue by Liquorice10113** [bottlesdevs/Bottles#3612]
### Describe the bug

So the exe won't launch by opening the exe in the file explorer and choosing bottles in the "open with" menu. The choose bottle runner dialog shows up, but after that there is the "UnboundLocalError: cannot access local variable 'program' where it is not associated with a value" in log and the exe doesn't run.

### To Reproduce

1. Go to file explorer (Nautilus in my case)
2. Open exe with bottles by double clicking and choosing bottles
3. In the choose bottle dialog, choose any.

### Package

Flatpak from Flathub

### Distribution

Ubuntu 24.04 LTS

### Debugging Information

```shell
Official Package: true
Version: '51.17'
DE/WM: ubuntu
Display:
    X.org: true
    X.org (port): :1
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json
            nvngx_path: /usr/lib/x86_64-linux-gnu/GL/nvidia-550-120/extra/nvidia/wine
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.8.0-51-generic
Disk:
    Total: 8299798528
    Free: 8299614208
RAM:
    MemTotal: 15.5GiB
    MemAvailable: 11.2GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
18:42:17 (INFO) Launching with URI: None 
18:42:17 (INFO) [Activate] request received. 
18:42:18 (INFO) Bottles Started! 
18:42:24 (INFO) Performing Bottles checks… 
18:42:24 (INFO) Dxvks found:
	 - dxvk-2.5
 
18:42:24 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.13
 
18:42:24 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.7.1
 
18:42:24 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1
 
18:42:24 (INFO) Runners found:
	 - soda-9.0-1
	 - wine-ge-proton8-26
	 - sys-wine-9.0
 
18:42:24 (INFO) Bottles found:
	 - WineGE
	 - soda
 
18:42:25 (INFO) Catalog installers loaded 
18:42:25 (INFO) Catalog dependencies loaded 
18:42:26 (INFO) Catalog components loaded 
18:42:30 (INFO) Launching with URI: ['/home/ki/WPS-Setup.exe'] 
18:42:30 (INFO) Bottles found:
	 - WineGE
	 - soda
 
18:42:31 (INFO) Forcing offline mode 
Traceback (most recent call last):
  File "/app/bin/bottles-cli", line 758, in <module>
    cli = CLI()
          ^^^^^
  File "/app/bin/bottles-cli", line 236, in __init__
    self.__process_args()
  File "/app/bin/bottles-cli", line 285, in __process_args
    self.run_program()
  File "/app/bin/bottles-cli", line 699, in run_program
    WineExecutor.run_program(bottle, program)
                                     ^^^^^^^
UnboundLocalError: cannot access local variable 'program' where it is not associated with a value
```

**Comment by Liquorice10113** [comment]
#3601

**Comment by DFaby26** [comment]
tengo entendido que ya esta solucionado solo que no se ha construido la nueva versión.

**Comment by jntesteves** [comment]
Fixed in #3605, will be in the next version.
